iterators/code/iterators.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+
+# Task 1
+
+def fasta_reader(path):
+    id_, seq = None, ''
+    fasta_content = []
+    with open(path) as file:
+        for line in file:
+            fasta_content.append(line.rstrip())
+    for line in fasta_content:
+        if line.startswith(">"):
+            if id_: yield (id_, seq)
+            id_, seq = line, ''
```

Comment:
Лучше на отдельной строке, так более понятно и чисто
```suggestion
            if id_:
                yield (id_, seq)
                id_, seq = line, ''
```

Reply:
Строку лучше наращивать через списки с последующим join